[2019-09-16 11:09:36][spiderbase.py][line:74][INFO] 脚本错误:460100051hkgxq      http://gxq.haikou.gov.cn/blog/open/list?channelId=12&limit=10&offset=0
[2019-09-16 11:09:36][spiderbase.py][line:75][INFO] Traceback (most recent call last):
  File "F:\spider_demo\zzh_v3\zzh\spiders\spiderbase.py", line 49, in parse
    exec(func, self.global_namespace)
  File "<string>", line 25
    print title,url,pdate
              ^
SyntaxError: Missing parentheses in call to 'print'. Did you mean print(title,url,pdate)?

[2019-09-16 11:30:35][spiderbase.py][line:64][INFO] 脚本错误[未获取数据,请修改此脚本]:410000103hnswht      http://hct.henan.gov.cn/zwdt/gggs/
[2019-09-16 11:30:35][spiderbase.py][line:65][INFO] NoneType: None

[2019-09-16 11:34:30][spiderbase.py][line:64][INFO] 脚本错误[未获取数据,请修改此脚本]:410000103hnswht      http://hct.henan.gov.cn/zwdt/gggs/
[2019-09-16 11:34:30][spiderbase.py][line:65][INFO] NoneType: None

[2019-09-16 11:34:45][spiderbase.py][line:64][INFO] 脚本错误[未获取数据,请修改此脚本]:410000103hnswht      http://hct.henan.gov.cn/zwdt/gggs/
[2019-09-16 11:34:45][spiderbase.py][line:65][INFO] NoneType: None

[2019-09-16 11:35:20][spiderbase.py][line:64][INFO] 脚本错误[未获取数据,请修改此脚本]:410000103hnswht      http://hct.henan.gov.cn/zwdt/gggs/
[2019-09-16 11:35:20][spiderbase.py][line:65][INFO] NoneType: None

[2019-09-16 11:37:00][spiderbase.py][line:64][INFO] 脚本错误[未获取数据,请修改此脚本]:410000103hnswht      http://hct.henan.gov.cn/zwdt/gggs/
[2019-09-16 11:37:00][spiderbase.py][line:65][INFO] NoneType: None

[2019-09-16 11:37:45][spiderbase.py][line:64][INFO] 脚本错误[未获取数据,请修改此脚本]:410000103hnswht      http://hct.henan.gov.cn/zwdt/gggs/
[2019-09-16 11:37:45][spiderbase.py][line:65][INFO] NoneType: None

[2019-09-16 11:46:20][spiderbase.py][line:74][INFO] 脚本错误:330000101zjskjt      http://www.zjkjt.gov.cn/html/node01/list0101/0101_1.html
[2019-09-16 11:46:20][spiderbase.py][line:75][INFO] Traceback (most recent call last):
  File "F:\spider_demo\zzh_v3\zzh\spiders\spiderbase.py", line 49, in parse
    exec(func, self.global_namespace)
  File "<string>", line 54
    next_page_url = []
                      ^
IndentationError: unindent does not match any outer indentation level

[2019-09-16 11:53:15][spiderbase.py][line:74][INFO] 脚本错误:320000101jsskxjst      http://kxjst.jiangsu.gov.cn/col/col48967/index.html
[2019-09-16 11:53:15][spiderbase.py][line:75][INFO] Traceback (most recent call last):
  File "F:\spider_demo\zzh_v3\zzh\spiders\spiderbase.py", line 49, in parse
    exec(func, self.global_namespace)
  File "<string>", line 32
    print title,url,pdate
              ^
SyntaxError: Missing parentheses in call to 'print'. Did you mean print(title,url,pdate)?

[2019-09-16 13:43:05][spiderbase.py][line:64][INFO] 脚本错误[未获取数据,请修改此脚本]:100000101gjkjglxxxtggfwpt      https://service.most.gov.cn/2015tztg_all/
[2019-09-16 13:43:05][spiderbase.py][line:65][INFO] NoneType: None

[2019-09-16 13:52:50][spiderbase.py][line:64][INFO] 脚本错误[未获取数据,请修改此脚本]:340000102ahsjyt      http://jyt.ah.gov.cn/30
[2019-09-16 13:52:50][spiderbase.py][line:65][INFO] NoneType: None

[2019-09-16 14:10:10][spiderbase.py][line:64][INFO] 脚本错误[未获取数据,请修改此脚本]:520000304gzsczt      http://czt.guizhou.gov.cn/xwzx/tzgg/
[2019-09-16 14:10:10][spiderbase.py][line:65][INFO] NoneType: None

[2019-09-16 14:15:30][spiderbase.py][line:74][INFO] 脚本错误:520000304gzsczt      http://czt.guizhou.gov.cn/xwzx/tzgg/
[2019-09-16 14:15:30][spiderbase.py][line:75][INFO] Traceback (most recent call last):
  File "F:\spider_demo\zzh_v3\zzh\spiders\spiderbase.py", line 49, in parse
    exec(func, self.global_namespace)
  File "<string>", line 52
    print next_page_url
                      ^
SyntaxError: Missing parentheses in call to 'print'. Did you mean print(next_page_url)?

[2019-09-16 14:24:55][spiderbase.py][line:64][INFO] 脚本错误[未获取数据,请修改此脚本]:520000901gzkxy       http://gzas.guizhou.gov.cn/xwzx/tzgg/
[2019-09-16 14:24:55][spiderbase.py][line:65][INFO] NoneType: None

[2019-09-16 14:25:50][spiderbase.py][line:64][INFO] 脚本错误[未获取数据,请修改此脚本]:520000901gzkxy       http://gzas.guizhou.gov.cn/xwzx/tzgg/
[2019-09-16 14:25:50][spiderbase.py][line:65][INFO] NoneType: None

[2019-09-16 14:33:25][spiderbase.py][line:64][INFO] 脚本错误[未获取数据,请修改此脚本]:620100051lzsgjgxjscykfq01      http://www.lzgxcy.com/List/5
[2019-09-16 14:33:25][spiderbase.py][line:65][INFO] NoneType: None

[2019-09-16 15:17:50][spiderbase.py][line:64][INFO] 脚本错误[未获取数据,请修改此脚本]:530000002ynsrmzf      http://www.yn.gov.cn/zwgk/zcwj/zxwj/
[2019-09-16 15:17:50][spiderbase.py][line:65][INFO] NoneType: None

[2019-09-16 15:22:50][spiderbase.py][line:64][INFO] 脚本错误[未获取数据,请修改此脚本]:460000101hnsxwqyml      http://xwqy.gsxt.gov.cn/mirco/info_list?organId=100000&channelId=1203
[2019-09-16 15:22:50][spiderbase.py][line:65][INFO] NoneType: None

[2019-09-16 15:46:10][spiderbase.py][line:74][INFO] 脚本错误:442000101zsskxjsj      http://www.zskj.gov.cn/home/list3Home.do?Q_status_SN_EQ=1&dir=DESC&sort=createTime&limit=10&searchAll=false&Q_globalTypes.nodeKey_S_EQ=Announcement&start=0
[2019-09-16 15:46:10][spiderbase.py][line:75][INFO] Traceback (most recent call last):
  File "F:\spider_demo\zzh_v3\zzh\spiders\spiderbase.py", line 51, in parse
    item_list, next_page_url = parse_spider(response, dept_id, dept_name_key)
  File "<string>", line 6, in parse_spider
  File "D:\Program Files\Python36\Lib\json\__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "D:\Program Files\Python36\Lib\json\decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "D:\Program Files\Python36\Lib\json\decoder.py", line 355, in raw_decode
    obj, end = self.scan_once(s, idx)
json.decoder.JSONDecodeError: Expecting property name enclosed in double quotes: line 4 column 3 (char 8)

[2019-09-16 15:51:05][spiderbase.py][line:74][INFO] 脚本错误:442000101zsskxjsj      http://www.zskj.gov.cn/home/list3Home.do?Q_status_SN_EQ=1&dir=DESC&sort=createTime&limit=10&searchAll=false&Q_globalTypes.nodeKey_S_EQ=Announcement&start=0
[2019-09-16 15:51:05][spiderbase.py][line:75][INFO] Traceback (most recent call last):
  File "F:\spider_demo\zzh_v3\zzh\spiders\spiderbase.py", line 49, in parse
    exec(func, self.global_namespace)
  File "<string>", line 33
    print title,url,pdate
              ^
SyntaxError: Missing parentheses in call to 'print'. Did you mean print(title,url,pdate)?

[2019-09-16 15:51:20][spiderbase.py][line:74][INFO] 脚本错误:442000101zsskxjsj      http://www.zskj.gov.cn/home/list3Home.do?Q_status_SN_EQ=1&dir=DESC&sort=createTime&limit=10&searchAll=false&Q_globalTypes.nodeKey_S_EQ=Announcement&start=0
[2019-09-16 15:51:20][spiderbase.py][line:75][INFO] Traceback (most recent call last):
  File "F:\spider_demo\zzh_v3\zzh\spiders\spiderbase.py", line 51, in parse
    item_list, next_page_url = parse_spider(response, dept_id, dept_name_key)
  File "<string>", line 5, in parse_spider
  File "D:\Program Files\Python36\Lib\json\__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "D:\Program Files\Python36\Lib\json\decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "D:\Program Files\Python36\Lib\json\decoder.py", line 355, in raw_decode
    obj, end = self.scan_once(s, idx)
json.decoder.JSONDecodeError: Expecting property name enclosed in double quotes: line 4 column 3 (char 8)

[2019-09-16 15:52:20][spiderbase.py][line:74][INFO] 脚本错误:442000101zsskxjsj      http://www.zskj.gov.cn/home/list3Home.do?Q_status_SN_EQ=1&dir=DESC&sort=createTime&limit=10&searchAll=false&Q_globalTypes.nodeKey_S_EQ=Announcement&start=0
[2019-09-16 15:52:20][spiderbase.py][line:75][INFO] Traceback (most recent call last):
  File "F:\spider_demo\zzh_v3\zzh\spiders\spiderbase.py", line 51, in parse
    item_list, next_page_url = parse_spider(response, dept_id, dept_name_key)
  File "<string>", line 5, in parse_spider
  File "D:\Program Files\Python36\Lib\json\__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "D:\Program Files\Python36\Lib\json\decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "D:\Program Files\Python36\Lib\json\decoder.py", line 355, in raw_decode
    obj, end = self.scan_once(s, idx)
json.decoder.JSONDecodeError: Expecting property name enclosed in double quotes: line 4 column 3 (char 8)

[2019-09-16 15:53:21][spiderbase.py][line:74][INFO] 脚本错误:442000101zsskxjsj      http://www.zskj.gov.cn/home/list3Home.do?Q_status_SN_EQ=1&dir=DESC&sort=createTime&limit=10&searchAll=false&Q_globalTypes.nodeKey_S_EQ=Announcement&start=0
[2019-09-16 15:53:21][spiderbase.py][line:75][INFO] Traceback (most recent call last):
  File "F:\spider_demo\zzh_v3\zzh\spiders\spiderbase.py", line 51, in parse
    item_list, next_page_url = parse_spider(response, dept_id, dept_name_key)
  File "<string>", line 7, in parse_spider
TypeError: string indices must be integers

[2019-09-16 15:54:15][spiderbase.py][line:74][INFO] 脚本错误:442000101zsskxjsj      http://www.zskj.gov.cn/home/list3Home.do?Q_status_SN_EQ=1&dir=DESC&sort=createTime&limit=10&searchAll=false&Q_globalTypes.nodeKey_S_EQ=Announcement&start=0
[2019-09-16 15:54:15][spiderbase.py][line:75][INFO] Traceback (most recent call last):
  File "F:\spider_demo\zzh_v3\zzh\spiders\spiderbase.py", line 51, in parse
    item_list, next_page_url = parse_spider(response, dept_id, dept_name_key)
  File "<string>", line 8, in parse_spider
TypeError: string indices must be integers

[2019-09-16 15:55:21][spiderbase.py][line:74][INFO] 脚本错误:442000101zsskxjsj      http://www.zskj.gov.cn/home/list3Home.do?Q_status_SN_EQ=1&dir=DESC&sort=createTime&limit=10&searchAll=false&Q_globalTypes.nodeKey_S_EQ=Announcement&start=0
[2019-09-16 15:55:21][spiderbase.py][line:75][INFO] Traceback (most recent call last):
  File "F:\spider_demo\zzh_v3\zzh\spiders\spiderbase.py", line 51, in parse
    item_list, next_page_url = parse_spider(response, dept_id, dept_name_key)
  File "<string>", line 5, in parse_spider
  File "D:\Program Files\Python36\Lib\json\__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "D:\Program Files\Python36\Lib\json\decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "D:\Program Files\Python36\Lib\json\decoder.py", line 355, in raw_decode
    obj, end = self.scan_once(s, idx)
json.decoder.JSONDecodeError: Expecting property name enclosed in double quotes: line 4 column 3 (char 8)

[2019-09-16 15:56:25][spiderbase.py][line:74][INFO] 脚本错误:442000101zsskxjsj      http://www.zskj.gov.cn/home/list3Home.do?Q_status_SN_EQ=1&dir=DESC&sort=createTime&limit=10&searchAll=false&Q_globalTypes.nodeKey_S_EQ=Announcement&start=0
[2019-09-16 15:56:25][spiderbase.py][line:75][INFO] Traceback (most recent call last):
  File "F:\spider_demo\zzh_v3\zzh\spiders\spiderbase.py", line 51, in parse
    item_list, next_page_url = parse_spider(response, dept_id, dept_name_key)
  File "<string>", line 5, in parse_spider
  File "D:\Program Files\Python36\Lib\json\__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "D:\Program Files\Python36\Lib\json\decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "D:\Program Files\Python36\Lib\json\decoder.py", line 355, in raw_decode
    obj, end = self.scan_once(s, idx)
json.decoder.JSONDecodeError: Expecting property name enclosed in double quotes: line 4 column 3 (char 8)

[2019-09-16 15:56:46][spiderbase.py][line:74][INFO] 脚本错误:442000101zsskxjsj      http://www.zskj.gov.cn/home/list3Home.do?Q_status_SN_EQ=1&dir=DESC&sort=createTime&limit=10&searchAll=false&Q_globalTypes.nodeKey_S_EQ=Announcement&start=0
[2019-09-16 15:56:46][spiderbase.py][line:75][INFO] Traceback (most recent call last):
  File "F:\spider_demo\zzh_v3\zzh\spiders\spiderbase.py", line 51, in parse
    item_list, next_page_url = parse_spider(response, dept_id, dept_name_key)
  File "<string>", line 8, in parse_spider
AttributeError: 'bytes' object has no attribute 'get'

[2019-09-16 15:57:15][spiderbase.py][line:74][INFO] 脚本错误:442000101zsskxjsj      http://www.zskj.gov.cn/home/list3Home.do?Q_status_SN_EQ=1&dir=DESC&sort=createTime&limit=10&searchAll=false&Q_globalTypes.nodeKey_S_EQ=Announcement&start=0
[2019-09-16 15:57:15][spiderbase.py][line:75][INFO] Traceback (most recent call last):
  File "F:\spider_demo\zzh_v3\zzh\spiders\spiderbase.py", line 51, in parse
    item_list, next_page_url = parse_spider(response, dept_id, dept_name_key)
  File "<string>", line 8, in parse_spider
AttributeError: 'str' object has no attribute 'get'

[2019-09-16 15:57:40][spiderbase.py][line:74][INFO] 脚本错误:442000101zsskxjsj      http://www.zskj.gov.cn/home/list3Home.do?Q_status_SN_EQ=1&dir=DESC&sort=createTime&limit=10&searchAll=false&Q_globalTypes.nodeKey_S_EQ=Announcement&start=0
[2019-09-16 15:57:40][spiderbase.py][line:75][INFO] Traceback (most recent call last):
  File "F:\spider_demo\zzh_v3\zzh\spiders\spiderbase.py", line 51, in parse
    item_list, next_page_url = parse_spider(response, dept_id, dept_name_key)
  File "<string>", line 5, in parse_spider
  File "D:\Program Files\Python36\Lib\json\__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "D:\Program Files\Python36\Lib\json\decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "D:\Program Files\Python36\Lib\json\decoder.py", line 355, in raw_decode
    obj, end = self.scan_once(s, idx)
json.decoder.JSONDecodeError: Expecting property name enclosed in double quotes: line 4 column 3 (char 8)

[2019-09-16 15:57:56][spiderbase.py][line:74][INFO] 脚本错误:442000101zsskxjsj      http://www.zskj.gov.cn/home/list3Home.do?Q_status_SN_EQ=1&dir=DESC&sort=createTime&limit=10&searchAll=false&Q_globalTypes.nodeKey_S_EQ=Announcement&start=0
[2019-09-16 15:57:56][spiderbase.py][line:75][INFO] Traceback (most recent call last):
  File "F:\spider_demo\zzh_v3\zzh\spiders\spiderbase.py", line 51, in parse
    item_list, next_page_url = parse_spider(response, dept_id, dept_name_key)
  File "<string>", line 5, in parse_spider
  File "D:\Program Files\Python36\Lib\json\__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "D:\Program Files\Python36\Lib\json\decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "D:\Program Files\Python36\Lib\json\decoder.py", line 355, in raw_decode
    obj, end = self.scan_once(s, idx)
json.decoder.JSONDecodeError: Expecting property name enclosed in double quotes: line 4 column 3 (char 8)

[2019-09-16 15:58:15][spiderbase.py][line:74][INFO] 脚本错误:442000101zsskxjsj      http://www.zskj.gov.cn/home/list3Home.do?Q_status_SN_EQ=1&dir=DESC&sort=createTime&limit=10&searchAll=false&Q_globalTypes.nodeKey_S_EQ=Announcement&start=0
[2019-09-16 15:58:15][spiderbase.py][line:75][INFO] Traceback (most recent call last):
  File "F:\spider_demo\zzh_v3\zzh\spiders\spiderbase.py", line 51, in parse
    item_list, next_page_url = parse_spider(response, dept_id, dept_name_key)
  File "<string>", line 5, in parse_spider
  File "D:\Program Files\Python36\Lib\json\__init__.py", line 231, in dumps
    return _default_encoder.encode(obj)
  File "D:\Program Files\Python36\Lib\json\encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "D:\Program Files\Python36\Lib\json\encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "D:\Program Files\Python36\Lib\json\encoder.py", line 180, in default
    o.__class__.__name__)
TypeError: Object of type 'bytes' is not JSON serializable

[2019-09-16 15:58:40][spiderbase.py][line:74][INFO] 脚本错误:442000101zsskxjsj      http://www.zskj.gov.cn/home/list3Home.do?Q_status_SN_EQ=1&dir=DESC&sort=createTime&limit=10&searchAll=false&Q_globalTypes.nodeKey_S_EQ=Announcement&start=0
[2019-09-16 15:58:40][spiderbase.py][line:75][INFO] Traceback (most recent call last):
  File "F:\spider_demo\zzh_v3\zzh\spiders\spiderbase.py", line 51, in parse
    item_list, next_page_url = parse_spider(response, dept_id, dept_name_key)
  File "<string>", line 5, in parse_spider
  File "D:\Program Files\Python36\Lib\json\__init__.py", line 231, in dumps
    return _default_encoder.encode(obj)
  File "D:\Program Files\Python36\Lib\json\encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "D:\Program Files\Python36\Lib\json\encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "D:\Program Files\Python36\Lib\json\encoder.py", line 180, in default
    o.__class__.__name__)
TypeError: Object of type 'method' is not JSON serializable

[2019-09-16 15:59:31][spiderbase.py][line:74][INFO] 脚本错误:442000101zsskxjsj      http://www.zskj.gov.cn/home/list3Home.do?Q_status_SN_EQ=1&dir=DESC&sort=createTime&limit=10&searchAll=false&Q_globalTypes.nodeKey_S_EQ=Announcement&start=0
[2019-09-16 15:59:31][spiderbase.py][line:75][INFO] Traceback (most recent call last):
  File "F:\spider_demo\zzh_v3\zzh\spiders\spiderbase.py", line 51, in parse
    item_list, next_page_url = parse_spider(response, dept_id, dept_name_key)
  File "<string>", line 8, in parse_spider
AttributeError: 'function' object has no attribute 'get'

[2019-09-16 16:00:10][spiderbase.py][line:74][INFO] 脚本错误:442000101zsskxjsj      http://www.zskj.gov.cn/home/list3Home.do?Q_status_SN_EQ=1&dir=DESC&sort=createTime&limit=10&searchAll=false&Q_globalTypes.nodeKey_S_EQ=Announcement&start=0
[2019-09-16 16:00:10][spiderbase.py][line:75][INFO] Traceback (most recent call last):
  File "F:\spider_demo\zzh_v3\zzh\spiders\spiderbase.py", line 51, in parse
    item_list, next_page_url = parse_spider(response, dept_id, dept_name_key)
  File "<string>", line 5, in parse_spider
  File "D:\Program Files\Python36\Lib\json\__init__.py", line 348, in loads
    'not {!r}'.format(s.__class__.__name__))
TypeError: the JSON object must be str, bytes or bytearray, not 'method'

[2019-09-16 16:00:50][spiderbase.py][line:74][INFO] 脚本错误:442000101zsskxjsj      http://www.zskj.gov.cn/home/list3Home.do?Q_status_SN_EQ=1&dir=DESC&sort=createTime&limit=10&searchAll=false&Q_globalTypes.nodeKey_S_EQ=Announcement&start=0
[2019-09-16 16:00:50][spiderbase.py][line:75][INFO] Traceback (most recent call last):
  File "F:\spider_demo\zzh_v3\zzh\spiders\spiderbase.py", line 51, in parse
    item_list, next_page_url = parse_spider(response, dept_id, dept_name_key)
  File "<string>", line 5, in parse_spider
  File "D:\Program Files\Python36\Lib\json\__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "D:\Program Files\Python36\Lib\json\decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "D:\Program Files\Python36\Lib\json\decoder.py", line 355, in raw_decode
    obj, end = self.scan_once(s, idx)
json.decoder.JSONDecodeError: Expecting property name enclosed in double quotes: line 4 column 3 (char 8)

[2019-09-16 16:01:15][spiderbase.py][line:74][INFO] 脚本错误:442000101zsskxjsj      http://www.zskj.gov.cn/home/list3Home.do?Q_status_SN_EQ=1&dir=DESC&sort=createTime&limit=10&searchAll=false&Q_globalTypes.nodeKey_S_EQ=Announcement&start=0
[2019-09-16 16:01:15][spiderbase.py][line:75][INFO] Traceback (most recent call last):
  File "F:\spider_demo\zzh_v3\zzh\spiders\spiderbase.py", line 51, in parse
    item_list, next_page_url = parse_spider(response, dept_id, dept_name_key)
  File "<string>", line 5, in parse_spider
AttributeError: 'bytes' object has no attribute 'encode'

[2019-09-16 16:01:30][spiderbase.py][line:74][INFO] 脚本错误:442000101zsskxjsj      http://www.zskj.gov.cn/home/list3Home.do?Q_status_SN_EQ=1&dir=DESC&sort=createTime&limit=10&searchAll=false&Q_globalTypes.nodeKey_S_EQ=Announcement&start=0
[2019-09-16 16:01:30][spiderbase.py][line:75][INFO] Traceback (most recent call last):
  File "F:\spider_demo\zzh_v3\zzh\spiders\spiderbase.py", line 51, in parse
    item_list, next_page_url = parse_spider(response, dept_id, dept_name_key)
  File "<string>", line 5, in parse_spider
  File "D:\Program Files\Python36\Lib\json\__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "D:\Program Files\Python36\Lib\json\decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "D:\Program Files\Python36\Lib\json\decoder.py", line 355, in raw_decode
    obj, end = self.scan_once(s, idx)
json.decoder.JSONDecodeError: Expecting property name enclosed in double quotes: line 4 column 3 (char 8)

[2019-09-16 16:05:45][spiderbase.py][line:74][INFO] 脚本错误:442000101zsskxjsj      http://www.zskj.gov.cn/home/list3Home.do?Q_status_SN_EQ=1&dir=DESC&sort=createTime&limit=10&searchAll=false&Q_globalTypes.nodeKey_S_EQ=Announcement&start=0
[2019-09-16 16:05:45][spiderbase.py][line:75][INFO] Traceback (most recent call last):
  File "F:\spider_demo\zzh_v3\zzh\spiders\spiderbase.py", line 51, in parse
    item_list, next_page_url = parse_spider(response, dept_id, dept_name_key)
  File "<string>", line 18, in parse_spider
  File "D:\zhengzihui\python-venv\spider-venv\lib\re.py", line 222, in findall
    return _compile(pattern, flags).findall(string)
TypeError: cannot use a string pattern on a bytes-like object

[2019-09-16 16:09:25][spiderbase.py][line:74][INFO] 脚本错误:440307101szslgqkjcxj01      http://www.lg.gov.cn/bmzz/kjj/xxgk/qt/tzgg/
[2019-09-16 16:09:25][spiderbase.py][line:75][INFO] Traceback (most recent call last):
  File "F:\spider_demo\zzh_v3\zzh\spiders\spiderbase.py", line 51, in parse
    item_list, next_page_url = parse_spider(response, dept_id, dept_name_key)
ValueError: too many values to unpack (expected 2)

[2019-09-16 16:12:05][spiderbase.py][line:64][INFO] 脚本错误[未获取数据,请修改此脚本]:440307101szslgqkjcxj01      http://www.lg.gov.cn/bmzz/kjj/xxgk/qt/tzgg/
[2019-09-16 16:12:05][spiderbase.py][line:65][INFO] NoneType: None

[2019-09-16 16:14:20][spiderbase.py][line:64][INFO] 脚本错误[未获取数据,请修改此脚本]:440305101szsnsqkjcxj      http://www.szns.gov.cn/xxgk/bmxxgk/qkcj/xxgk/qt/tzgg/
[2019-09-16 16:14:20][spiderbase.py][line:65][INFO] NoneType: None

[2019-09-16 16:23:55][spiderbase.py][line:64][INFO] 脚本错误[未获取数据,请修改此脚本]:440305004szsnsqrmzf      http://www.szns.gov.cn/xxgk/qzfxxgkml/tzgg/
[2019-09-16 16:23:55][spiderbase.py][line:65][INFO] NoneType: None

[2019-09-16 16:45:25][spiderbase.py][line:64][INFO] 脚本错误[未获取数据,请修改此脚本]:100000106zhrmghggjzscqj      http://www.cnipa.gov.cn/gztz/index.htm
[2019-09-16 16:45:25][spiderbase.py][line:65][INFO] NoneType: None

[2019-09-16 17:10:25][spiderbase.py][line:64][INFO] 脚本错误[未获取数据,请修改此脚本]:130100301sjzsgyhxxhj      http://gxj.sjz.gov.cn/col/1506393935681/index.html
[2019-09-16 17:10:25][spiderbase.py][line:65][INFO] NoneType: None

[2019-09-16 17:28:10][spiderbase.py][line:74][INFO] 脚本错误:320582101zjgskjj      http://www.zjg.gov.cn/AjaxJsonData/govpub_e393ad73-fb93-447b-8555-607332958035-0.json
[2019-09-16 17:28:10][spiderbase.py][line:75][INFO] Traceback (most recent call last):
  File "F:\spider_demo\zzh_v3\zzh\spiders\spiderbase.py", line 51, in parse
    item_list, next_page_url = parse_spider(response, dept_id, dept_name_key)
  File "<string>", line 5, in parse_spider
  File "D:\Program Files\Python36\Lib\json\__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "D:\Program Files\Python36\Lib\json\decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "D:\Program Files\Python36\Lib\json\decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

[2019-09-16 17:28:45][spiderbase.py][line:74][INFO] 脚本错误:320582101zjgskjj      http://www.zjg.gov.cn/AjaxJsonData/govpub_e393ad73-fb93-447b-8555-607332958035-0.json
[2019-09-16 17:28:45][spiderbase.py][line:75][INFO] Traceback (most recent call last):
  File "F:\spider_demo\zzh_v3\zzh\spiders\spiderbase.py", line 51, in parse
    item_list, next_page_url = parse_spider(response, dept_id, dept_name_key)
  File "<string>", line 9, in parse_spider
TypeError: string indices must be integers

[2019-09-16 17:29:25][spiderbase.py][line:74][INFO] 脚本错误:320582101zjgskjj      http://www.zjg.gov.cn/AjaxJsonData/govpub_e393ad73-fb93-447b-8555-607332958035-0.json
[2019-09-16 17:29:25][spiderbase.py][line:75][INFO] Traceback (most recent call last):
  File "F:\spider_demo\zzh_v3\zzh\spiders\spiderbase.py", line 51, in parse
    item_list, next_page_url = parse_spider(response, dept_id, dept_name_key)
  File "<string>", line 5, in parse_spider
  File "D:\Program Files\Python36\Lib\json\__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "D:\Program Files\Python36\Lib\json\decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "D:\Program Files\Python36\Lib\json\decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

[2019-09-16 17:29:45][spiderbase.py][line:74][INFO] 脚本错误:320582101zjgskjj      http://www.zjg.gov.cn/AjaxJsonData/govpub_e393ad73-fb93-447b-8555-607332958035-0.json
[2019-09-16 17:29:45][spiderbase.py][line:75][INFO] Traceback (most recent call last):
  File "F:\spider_demo\zzh_v3\zzh\spiders\spiderbase.py", line 51, in parse
    item_list, next_page_url = parse_spider(response, dept_id, dept_name_key)
  File "<string>", line 9, in parse_spider
TypeError: string indices must be integers

[2019-09-16 17:35:25][spiderbase.py][line:64][INFO] 脚本错误[未获取数据,请修改此脚本]:320582101zjgskjj      http://www.zjg.gov.cn/AjaxJsonData/govpub_e393ad73-fb93-447b-8555-607332958035-0.json
[2019-09-16 17:35:25][spiderbase.py][line:65][INFO] NoneType: None

[2019-09-17 09:01:19][spiderbase.py][line:64][INFO] 脚本错误[未获取数据,请修改此脚本]:320582101zjgskjj      http://www.zjg.gov.cn/AjaxJsonData/govpub_e393ad73-fb93-447b-8555-607332958035-0.json
[2019-09-17 09:01:19][spiderbase.py][line:65][INFO] NoneType: None

[2019-09-17 09:03:34][spiderbase.py][line:64][INFO] 脚本错误[未获取数据,请修改此脚本]:320582101zjgskjj      http://www.zjg.gov.cn/AjaxJsonData/govpub_e393ad73-fb93-447b-8555-607332958035-0.json
[2019-09-17 09:03:34][spiderbase.py][line:65][INFO] NoneType: None

[2019-09-17 09:03:59][spiderbase.py][line:64][INFO] 脚本错误[未获取数据,请修改此脚本]:320582101zjgskjj      http://www.zjg.gov.cn/AjaxJsonData/govpub_e393ad73-fb93-447b-8555-607332958035-0.json
[2019-09-17 09:03:59][spiderbase.py][line:65][INFO] NoneType: None

[2019-09-17 09:05:39][spiderbase.py][line:74][INFO] 脚本错误:320582101zjgskjj      http://www.zjg.gov.cn/AjaxJsonData/govpub_e393ad73-fb93-447b-8555-607332958035-0.json
[2019-09-17 09:05:39][spiderbase.py][line:75][INFO] Traceback (most recent call last):
  File "F:\spider_demo\zzh_v3\zzh\spiders\spiderbase.py", line 51, in parse
    item_list, next_page_url = parse_spider(response, dept_id, dept_name_key)
  File "<string>", line 11, in parse_spider
  File "D:\zhengzihui\python-venv\spider-venv\lib\re.py", line 222, in findall
    return _compile(pattern, flags).findall(string)
TypeError: cannot use a string pattern on a bytes-like object

[2019-09-17 09:08:09][spiderbase.py][line:74][INFO] 脚本错误:320582101zjgskjj      http://www.zjg.gov.cn/AjaxJsonData/govpub_e393ad73-fb93-447b-8555-607332958035-0.json
[2019-09-17 09:08:09][spiderbase.py][line:75][INFO] Traceback (most recent call last):
  File "F:\spider_demo\zzh_v3\zzh\spiders\spiderbase.py", line 51, in parse
    item_list, next_page_url = parse_spider(response, dept_id, dept_name_key)
  File "<string>", line 11, in parse_spider
  File "D:\zhengzihui\python-venv\spider-venv\lib\re.py", line 222, in findall
    return _compile(pattern, flags).findall(string)
TypeError: cannot use a string pattern on a bytes-like object

[2019-09-17 10:21:33][spiderbase.py][line:65][INFO] 脚本错误[未获取数据,请修改此脚本]:440103108gzslwqkgsxj      http://www.lw.gov.cn/lwqkxj/002002/xxgk_list_1.shtml
[2019-09-17 10:21:33][spiderbase.py][line:66][INFO] NoneType: None

[2019-09-17 10:24:58][spiderbase.py][line:65][INFO] 脚本错误[未获取数据,请修改此脚本]:440103201gzslwqswnyj_gzslwqwnb_gzslwqcmj      http://www.lw.gov.cn/lwqswnyj/002002/xxgk_list_1.shtml
[2019-09-17 10:24:58][spiderbase.py][line:66][INFO] NoneType: None

[2019-09-17 10:34:18][spiderbase.py][line:75][INFO] 脚本错误:440104004gzsyxqrmzf      http://www.yuexiu.gov.cn/yxxxw/pc/yxdt/tzgg_yxdt/index.shtml
[2019-09-17 10:34:18][spiderbase.py][line:76][INFO] Traceback (most recent call last):
  File "F:\spider_demo\zzh_v3\zzh\spiders\spiderbase.py", line 50, in parse
    exec(func, self.global_namespace)
  File "<string>", line 52
    print next_page_url
                      ^
SyntaxError: Missing parentheses in call to 'print'. Did you mean print(next_page_url)?

[2019-09-17 10:58:36][spiderbase.py][line:65][INFO] 脚本错误[未获取数据,请修改此脚本]:420100302whsjjhxxhwyh      http://jxw.wuhan.gov.cn/ArticleList.aspx?cid=10
[2019-09-17 10:58:36][spiderbase.py][line:66][INFO] NoneType: None

[2019-09-17 11:01:51][spiderbase.py][line:65][INFO] 脚本错误[未获取数据,请修改此脚本]:420100302whsjjhxxhwyh      http://jxw.wuhan.gov.cn/ArticleList.aspx?cid=10&y7bRbP=8BnxcacxMMUSl3i_IHNECSnvcODUoqYbGQZUEcdFrPAqqJE
[2019-09-17 11:01:51][spiderbase.py][line:66][INFO] NoneType: None

[2019-09-17 11:02:16][spiderbase.py][line:65][INFO] 脚本错误[未获取数据,请修改此脚本]:420100302whsjjhxxhwyh      http://jxw.wuhan.gov.cn/ArticleList.aspx?cid=10&y7bRbP=8BnxcacxMMUSl3i_IHNECSnvcODUoqYbGQZUEcdFrPAqqJE
[2019-09-17 11:02:16][spiderbase.py][line:66][INFO] NoneType: None

